src/test/java/com/epam/tc/hw7/utils/ReaderUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.tc.hw7.utils;
+
+import java.util.List;
+import java.util.ListIterator;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class ReaderUtil {
+
+    public static String getString(List<String> objectList) {
+        StringBuilder result = new StringBuilder();
+        ListIterator<String> iterator = objectList.listIterator();
+
+        for (String str : objectList) {
+            result.append(str);
+            iterator.next();
+
+            if (iterator.hasNext()) {
```

Comment:
Why this code is so difficult? All this code is equal to this string: String result = String.join(",", objectList);